equalsverifier-core/src/test/java/nl/jqno/equalsverifier/internal/reflection/FieldIterableTest.java

```diff
@@ -59,7 +59,7 @@ void onlySubClassFields() {
     @Test
     void noFields() {
         FieldIterable iterable = FieldIterable.of(NoFields.class);
-        assertThat(iterable.iterator().hasNext()).isFalse();
+        assertThat(iterable.iterator()).isExhausted();
```

Comment:
Again; we're being more expressive about what we expect, such that you get better failure messages.